modules/web/src/app/core/components/sidenav/component.ts

```diff
@@ -112,6 +113,9 @@ export class SidenavComponent implements OnInit, OnDestroy {
     const selectedProjectID = this._selectedProject.id;
     const urlArray = this._router.routerState.snapshot.url.split('/');
     const isProjectAndUrlExists = !!urlArray.find(x => x === selectedProjectID) && !!urlArray.find(x => x === url);
+    if (url === View.ClusterTemplates) {
+      return isProjectAndUrlExists || this._activatedRoute.snapshot.queryParams?.clusterTemplateWizard;
```

Comment:
I think we should also update this condition. Wdyt?
```suggestion
      const mode = this._activatedRoute.snapshot.queryParams?.mode;
      return isProjectAndUrlExists && !!urlArray.find(x => x === View.Wizard) && (mode === WizardMode.CreateClusterTemplate || mode === WizardMode.EditClusterTemplate || mode === WizardMode.CustomizeClusterTemplate);
```